README.md

```diff
@@ -1,8 +1,307 @@
+>>> API listening in host port 1239
+>>> Validator listening in host port 1349
+```
+> Beware: The validator address specified here should be the same as the one that will be used in the next step to join the subnet.
+
+This log provides information about the API and auth tokens for the daemon, default validator wallet used, the multiaddresses where the validator is listening, etc. To configure our IPC agent with this subnet daemon, we need to once again update our IPC agent with the relevant information. In this case, for the sample execution above we need to add the following section to the end of our config file:
+```toml
+[subnets."/root/t01002"]
+id = "/root/t01002"
+jsonrpc_api_http = "http://127.0.0.1:1239/rpc/v1"
+jsonrpc_api_ws = "wss://example.org/rpc/v0"
+auth_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.TnoDqZJ1fqdkr_oCHFEXvdwU6kYR7Va_ALyEuoPnksA"
+accounts = ["t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq"]
+```
+As always, remember to run `./bin/ipc_agent reload-config` for changes in the config of the agent to be picked up by the daemon.
+
+#### Joining a subnet
+With the daemon for the subnet deployed, we can join the subnet:
+```bash
+$ ./bin/ipc_agent join-subnet --subnet=<subnet-id> --collateral=<collateral_amount> --validator-net-addr=<libp2p-add-validator>
+
+# Sample execution
+$ ./bin/ipc_agent join-subnet --subnet=/root/t01002 --collateral=2 --validator-net-addr="GET_ADDRESS_FROM_SCRIPT"
+```
+This command specifies the subnet to join, the amount of collateral to provide and the validator net address used by other validators to dial them. We can pick up this information from the execution of the script above or running `eudico mir validator config validator-addr` from your deployment. Bear in mind that the multiaddress provided for the validator needs to be accessible publicly by other validators. According to the deployment used you may need to tweak the IP addresses of the multiaddresses and the ones provided by these scripts and commands won't be usable out-of-the-box.
+
+For instance, in the example above, we are using the DNS endpoint `/dns/host.docker.internal/` so other Docker containers for the subnet deployed in the host machine know how to contact the validator.
+
+As a sanity-check that we have joined the subnet successfully and that we provided enough collateral to register the subnet to IPC, we can list the child subnets of our parent with the following command:
+```bash
+
+$ ./bin/ipc_agent list-subnets --gateway-address=<gateway-addr> --subnet=<parent-subnet-id>
+
+# Sample execution
+$ ./bin/ipc_agent list-subnets --gateway-address=t064 --subnet=/root
+
+[2023-03-22T15:42:22Z INFO  ipc_agent::cli::commands::manager::list_subnets] found child subnets: {"/root/t01002": SubnetInfoWrapper { id: "/root/t01002", stake: 2000000000000000000, circ_supply: 0, status:
```

Comment:
So we join the subnet with colateral 2 but then the stake here is 2000000000000000000. Is this expressed in atofils and does it need to be? 🤔

Reply:
Correct, it is in attoFIL. I managed to include a quick fix so it is printed in FIL and it is clearer. Didn't do a proper PR to move fast, will be include in this same one.